holmes/plugins/toolsets/mongo.py

```
@@ -0,0 +1,270 @@
+import os
+import json
+import logging
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)